config/environments/test.rb

```diff
@@ -54,15 +54,4 @@
   # RackSessionAccess gives us access to the rack session
   # during feature tests
   config.middleware.use RackSessionAccess::Middleware
-
```

Comment:
This doesn't work well enough to keep: a bunch of system tests fail when Rails and Chrome disagree about what time it is. One can still do this when troubleshooting a particular test or whatever, but it's misleading for me to leave this here implying it'll just work.